controllers/tasks.js

```diff
@@ -112,6 +112,20 @@ const getSelfTasks = async (req, res) => {
     return res.boom.badImplementation("An internal server error occurred");
   }
 };
+
+const getTask = async (req, res) => {
+  try {
+    const taskId = req.params.id;
+    const { taskData } = await tasks.fetchTask(taskId);
+
+    if (!taskData) {
+      return res.boom.notFound("Task not found");
+    }
+    return res.json({ message: "task returned successfully", taskData });
```

Comment:
Is 'taskData' reduntant. Can it just be `data`?

Reply:
Yes, this can be changed, I did it because the function in the tasks model was returning `taskData` from there.

Reply:
this will be fixed in #736